src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDurationMappingTest.java

```diff
@@ -0,0 +1,371 @@
+    }, nullValues = {"null"})
+    void getHearingDurationBenefitIssueCodesPaper(String benefitCode, String issueCode) {
+
+        given(hearingDurations.getHearingDuration(benefitCode, issueCode)).willReturn(null);
+
+        given(referenceDataServiceHolder.getHearingDurations()).willReturn(hearingDurations);
+
+        SscsCaseData caseData = SscsCaseData.builder()
+            .benefitCode(benefitCode)
+            .issueCode(issueCode)
+            .appeal(Appeal.builder()
+                        .hearingSubtype(HearingSubtype.builder().build())
+                        .hearingOptions(HearingOptions.builder().build())
+                        .build())
+            .build();
+
+        Integer result = HearingsDurationMapping.getHearingDurationBenefitIssueCodes(
+            caseData,
+            referenceDataServiceHolder
+        );
+
+        assertThat(result).isNull();
+    }
+
+    @DisplayName("When wantsToAttend for the Appeal is null and the hearing type is paper "
+        + "getHearingDurationBenefitIssueCodes return the correct paper durations")
+    @Test
+    void getHearingDurationBenefitIssueCodesPaper() {
+
+        given(hearingDurations.getHearingDuration(BENEFIT_CODE, ISSUE_CODE))
+            .willReturn(new HearingDuration(BenefitCode.PIP_NEW_CLAIM, Issue.DD,
+                                            60, 75, 30
+            ));
+
+        given(referenceDataServiceHolder.getHearingDurations()).willReturn(hearingDurations);
+
+        SscsCaseData caseData = SscsCaseData.builder()
+            .benefitCode(BENEFIT_CODE)
+            .issueCode(ISSUE_CODE)
+            .appeal(Appeal.builder()
+                        .hearingSubtype(HearingSubtype.builder().build())
+                        .hearingOptions(HearingOptions.builder()
+                                            .wantsToAttend("No")
+                                            .build())
+                        .build())
+            .build();
+
+        Integer result = HearingsDurationMapping.getHearingDurationBenefitIssueCodes(
+            caseData,
+            referenceDataServiceHolder
+        );
+
+        assertThat(result).isEqualTo(30);
+    }
+
+    @DisplayName("When wantsToAttend for the Appeal is Yes and languageInterpreter is null "
+        + "getHearingDurationBenefitIssueCodes return the correct face to face durations")
+    @Test
+    void getHearingDurationBenefitIssueCodesFaceToFace() {
+        given(hearingDurations.getHearingDuration(BENEFIT_CODE, ISSUE_CODE))
+            .willReturn(new HearingDuration(BenefitCode.PIP_NEW_CLAIM, Issue.DD,
+                                            60, 75, 30
+            ));
+
+        given(hearingDurations.addExtraTimeIfNeeded(eq(60), eq(BenefitCode.PIP_NEW_CLAIM), eq(Issue.DD), any()))
+            .willReturn(60);
+
+        given(referenceDataServiceHolder.getHearingDurations()).willReturn(hearingDurations);
+
+        SscsCaseData caseData = SscsCaseData.builder()
```

Comment:
This builder and the one above can be extracted to a helper with a single wantsToAttend arg